go/chat/boxer_test.go

```diff
@@ -95,7 +95,7 @@ func getSigningKeyPairForTest(t *testing.T, tc *kbtest.ChatTestContext, u *kbtes
 			t.Fatal(err)
 		}
 	}
-	kp, err := tc.G.Keyrings.GetSecretKeyWithPassphrase(nil, u.User, u.Passphrase, nil)
+	kp, err := tc.G.Keyrings.GetSecretKeyWithPassphrase(kbtest.NewMetaContextForTest(*tc), u.User, u.Passphrase, nil)
```

Comment:
Instead of this `NewMetaContextForTest` function, should we just put a `MetaContext` on `tc` instead of `tc.G`? Is that too big of a change?

Reply:
hmm, that's fine unless you never want an actual `context.Context` to do any lifting in test. Maybe you don't?